CHANGELOG.md

```diff
@@ -44,37 +47,23 @@ This name should be decided amongst the team before the release.
 * [#533](https://github.com/tweag/topiary/pull/533) Update tree-sitter-ocaml to 0.20.3
 * [#535](https://github.com/tweag/topiary/pull/535) Improved error message when idempotency fails due to invalid output in the first pass.
 
-## [0.2.3] - 2023-06-20
+## v0.2.3 - Cyclic Cypress
 [0.2.2]: https://github.com/tweag/topiary/compare/v0.2.2...v0.2.3
```

Comment:
By removing the links to the GitHub comparison, these links are no longer referenced from anywhere, so they're redundant. Do we want to keep exposing them somehow?

(n.b., It looks like the markdown link tag for the 0.2.3 release isn't correct, here, but the general point still stands.)

Reply:
I don't mind this too much actually. The links are still there for those editing the file.

Reply:
We should be able to add a link back, as long as the braces aren't around the version number.

Reply:
This has been improved now.